src/NuGet.Clients/NuGet.Credentials/SecurePluginCredentialProvider.cs

```diff
@@ -91,6 +91,7 @@ public SecurePluginCredentialProvider(IPluginManager pluginManager, PluginDiscov
 
             if (_isAnAuthenticationPlugin)
             {
+                AddOrUpdateLogger(plugin.Plugin, _logger);
```

Comment:
why is this being passed as a function param to a private function if it's a class member?

Reply:
good point, updated it.